funcs/base64_gen.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by gencel. DO NOT EDIT.
+
+package funcs
+
+import "github.com/google/cel-go/cel"
+import "github.com/google/cel-go/common/types"
+import "github.com/google/cel-go/common/types/ref"
+
+var base64EncodeGen = cel.Function("base64.Encode",
+	cel.Overload("base64.Encode_interface{}",
```

Comment:
This can come out cel-ext already has base64.encode/deocde - Maybe alias to the cel names for gotemplate funcs as well

Reply:
added exclusions